spatial-tracking-explainer.md

```diff
@@ -349,24 +349,21 @@ partial interface XRSession {
 partial interface XRFrame {
   // Also listed in the main explainer.md
   XRViewerPose? getViewerPose(optional XRReferenceSpace referenceSpace);
-
-  // Also listed in input-explainer.md
-  XRInputPose? getInputPose(XRInputSource inputSource, optional XRReferenceSpace referenceSpace);
+  XRPose? getPose(XRSpace space, XRSpace relativeTo);
 };
 
 [SecureContext, Exposed=Window]
-interface XRInputPose {
+interface XRPose {
+  readonly attribute XRRigidTransform transform;
   readonly attribute boolean emulatedPosition;
-  readonly attribute XRRay targetRay;
-  readonly attribute XRRigidTransform? gripTransform;
 };
 
 //
 // Space
 //
 
 [SecureContext, Exposed=Window] interface XRSpace : EventTarget {
-  XRRigidTransform? getTransformTo(XRSpace other);
+  // Intentionally an opaque type, used by XRInputSource.
```

Comment:
"used by XRFrame"?

Reply:
No, this was meant to imply that the type was used as an attribute on XRInputSource, but I can see why there was some confusion. I'll simplify it to say just that it's intentionally opaque.